Plugin~/WebRTCPlugin/WebRTCPlugin.cpp

```diff
@@ -504,7 +562,7 @@ extern "C"
     }
 
     UNITY_INTERFACE_EXPORT RtpTransceiverInterface* PeerConnectionAddTransceiverWithTypeAndInit(
-        Context* context, PeerConnectionObject* obj, cricket::MediaType type, RtpTransceiverInit* init)
+        Context* context, PeerConnectionObject* obj, cricket::MediaType type, const RTCRtpTransceiverInit* init)
```

Comment:
https://github.com/Unity-Technologies/com.unity.webrtc/pull/746/files#diff-fc2c41712e2c2eab853d4dc3ce61e324c29cd2413aec2989c532aa1e9054c3afR544-R545
I think need to change argument `RtpTransceiverInit` -> `RTCRtpTransceiverInit` too.